src/heitech.zer0mqXt.core/Adapters/InternalAdapter.cs

```diff
@@ -0,0 +1,33 @@
+using System.Text;
+using heitech.zer0mqXt.core.infrastructure;
+
+namespace heitech.zer0mqXt.core.Adapters
+{
+    ///<summary>
+    /// Uses the default implementations of all services
+    ///</summary>
+    public class InternalAdapters : ISerializerAdapter
+    {
+        public Encoding Encoding { get; } = Encoding.UTF8;
+        private readonly Serializer _serializer;
+        private readonly BasicLogger _logger;
```

Comment:
`_logger` is currently unused. Can this be removed or is it intended to be used later on?

Reply:
no I changed it, as explained in another comment, and it will be removed